tap_adroll/client.py

```diff
@@ -74,7 +74,8 @@ def _make_request(self, method, endpoint, headers=None, params=None, data=None,
 
         # TODO: We should merge headers with some default headers like user_agent
         response = self.session.request(method, full_url, headers=headers, params=params, data=data)
-
+        if response.status_code == 403:
+            print(response.text)
```

Comment:
The description "Change circle config to run tests 3 times per day" does not match this code change?

Reply:
Yikes I pushed the wrong file. Removed bad change, pushed up correct one